pangeo_forge_recipes/storage.py

```diff
@@ -181,6 +181,10 @@ def getitems(self, keys: Sequence[str]) -> dict:
         return {k: json.loads(raw_bytes) for k, raw_bytes in all_meta_raw.items()}
 
 
+# A note about something frustrating about this context manager:
+# Sometimes it yields an fsspec.OpenFile object
+# Other times it yields something lower level, like an _io.BufferedReader
+# This can lead to unpredictable behavior
```

Comment:
This comment is outdated and needs to be removed.

Reply:
```suggestion

```